pkg/syncer/metadatasyncer.go

```diff
@@ -253,17 +261,71 @@ func InitMetadataSyncer(ctx context.Context, clusterFlavor cnstypes.CnsClusterFl
 	fullSyncTicker := time.NewTicker(time.Duration(getFullSyncIntervalInMin(ctx)) * time.Minute)
 	defer fullSyncTicker.Stop()
 	// Trigger full sync
-	go func() {
-		for ; true; <-fullSyncTicker.C {
-			ctx, log = logger.GetNewContextWithLogger()
-			log.Infof("fullSync is triggered")
-			if metadataSyncer.clusterFlavor == cnstypes.CnsClusterFlavorGuest {
-				pvcsiFullSync(ctx, metadataSyncer)
-			} else {
-				csiFullSync(ctx, metadataSyncer)
-			}
+	// If TriggerCsiFullSync feature gate is enabled, use TriggerCsiFullSync to trigger
+	// full sync. If not, directly invoke full sync methods.
+	if metadataSyncer.coCommonInterface.IsFSSEnabled(ctx, common.TriggerCsiFullSync) {
+		log.Infof("%q feature flag is enabled. Using TriggerCsiFullSync API to trigger full sync",
```

Comment:
I think this log can be debugf, since else{} is having a infof

Reply:
We want to have this log explicitly so that we know it is using the TriggerCsiFullSync API. It is a one time log in its lifetime.